packages/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "tabWidth": 4,
```

Comment:
```suggestion
    "tabWidth": 4,
```
is 4 not too much? I'd suggest two but it's subjective.

Reply:
It's for Move code.